sources/iTermPowerManager.h

```diff
@@ -12,7 +12,8 @@ extern NSString *const iTermPowerManagerStateDidChange;
 
 @interface iTermPowerManager : NSObject
 
-@property (nonatomic, readonly) BOOL connectedToPower;
+@property (nonatomic, assign) BOOL connectedToPower;
```

Comment:
Keep this readonly. In the .m file add:

````
@interface iTermPowerManager()
@property (nonatomic, readwrite) BOOL connectedToPower;
@end
````

This prevents external callers from changing the value.